src/libs/share/shareApp/__tests__/ShareAppModal.web.test.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+
+import { openUrl } from 'features/navigation/helpers/openUrl'
+import { env } from 'libs/environment'
+import { shareAppContent } from 'libs/share/shareApp/shareAppContent'
+import { ShareAppModal } from 'libs/share/shareApp/ShareAppModal'
+import { fireEvent, render } from 'tests/utils/web'
+
+const dismissModal = jest.fn()
+
+jest.mock('features/navigation/helpers/openUrl')
+const mockedOpenUrl = openUrl as jest.MockedFunction<typeof openUrl>
+
+const url = env.PASSCULTURE_DOWNLOAD_APP_URL
+const message = shareAppContent.message
+
+describe('<ShareAppModal />', () => {
+  it('should match snapshot', () => {
+    const renderAPI = render(<ShareAppModal visible={true} dismissModal={dismissModal} />)
+    expect(renderAPI).toMatchSnapshot()
+  })
+
+  it('should not display if visible false', () => {
+    const renderAPI = render(<ShareAppModal visible={false} dismissModal={dismissModal} />)
+    expect(renderAPI).not.toBeInTheDocument
```

Comment:
De même ici, pour un test web tu peux faire ça :
```suggestion
    expect(renderAPI.container).toBeEmptyDOMElement()
```